terraform/nomad-instance-pool/create-nomad-gpu-pool-stack.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -x 
+unset SSH_USER
+
+[ -z "$POOL_TYPE" ] && export POOL_TYPE="skynet"
+[ -z "$MEMORY_IN_GBS" ] && export MEMORY_IN_GBS="240"
+[ -z "$SHAPE" ] && export SHAPE="VM.GPU.A10.1"
+[ -z "$OCPUS" ] && export OCPUS="15"
```

Comment:
This is missing the disk size
```suggestion
[ -z "$OCPUS" ] && export OCPUS="15"
[ -z "$DISK_IN_GBS" ] && DISK_IN_GBS="100"
```